### DISCUSSION TITLE
How do I fix this error?

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [jpanther/lynx#134] [Q&A]
WARN  found no layout file for "html" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

I'm installing the theme as a go module

**Comment by SamGreenwood1** [comment]
another question: is there a way to assign custom names to buttons?

**Reply by dachico** [reply]
Hey Sam, assuming you mean HTML elements you can add a class or an id to them like that:
 <button class="custom">Text</button>,
or else you could add an id like that:
 <button id="custom">Text</button>,
Then in your CSS file you can target your class name with a dot like that:
.custom { In here apply your styles }
and to target an id you can use the # symbol instead of the dot

**Reply by SamGreenwood1** [reply]
So if I want to change the text I a button I have to create a completely new element every time, is that correct?

**Comment by SamGreenwood1** [comment]
> WARN  found no layout file for "html" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
> 
> I'm installing the theme as a go module

@dachico do you have an answer for my original question?
